File: FragmentExample/app/src/main/java/com/example/mypc/fragmentexample/MainActivity.java
package com.example.mypc.fragmentexample;

import android.app.Activity;

import android.support.v4.app.FragmentManager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Boolean status = false;
    Button bn; //creating variable for button

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        bn = (Button) findViewById(R.id.bn); //initializing variable bn
        bn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

               /* FragmentOne fragmentOne = new FragmentOne();
                FragmentManager manager = getSupportFragmentManager();
                manager.beginTransaction().replace(R.id.mainLayout, fragmentOne).commit();
                */

//                FragmentManager fragmentManager = getFragmentManager();
//                FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();

//                if(!status)
//
////                 FragmentOne f1 = new FragmentOne();
////                    fragmentTransaction.add(R.id.fragment_container,f1);
////                        fragmentTransaction.addToBackStack(null);
////                        fragmentTransaction.commit();
////                        bn.setText("LOAD SECOND FRAGMENT");
////                        status = true;
//                }
//                else
//                {
//                    FragmentTwo f2 = new FragmentTwo();
//                    fragmentTransaction.add(R.id.fragment_container,f2);
//                    fragmentTransaction.addToBackStack(null);
//                    fragmentTransaction.commit();
//                    bn.setText("LOAD FIRST FRAGMENT");
//                    status = false;
//                }
            }
        });
    }
}

File: UDEMY/CoderSwag/app/src/main/java/com/example/mypc/coderswag/Controller/MainActivity.kt
package com.example.mypc.coderswag.Controller

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.ArrayAdapter
import com.example.mypc.coderswag.Model.Category
import com.example.mypc.coderswag.R
import com.example.mypc.coderswag.Services.DataService
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    lateinit var adapter : ArrayAdapter<Category>

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //initializing an array adapter
        adapter = ArrayAdapter(this,
                android.R.layout.simple_list_item_1,
                DataService.categories)

        //set a adapter for a list view itself
        categoryListView.adapter = adapter
    }
}
